util/buildRelease/gen_release

```diff
@@ -134,6 +134,9 @@ if (exists($ENV{"CHPL_GEN_RELEASE_NO_CLONE"})) {
        "util/setchplenv.sh",
        "util/start_test",
        "util/chpltags",
+       "compiler/dyno/tools/chpldoc/LICENSE",
+       "compiler/dyno/tools/chpldoc/BUILD_VERSION",
+       "compiler/dyno/tools/chpldoc/COPYRIGHT",
```

Comment:
Do these files exist?  I'm not seeing them in my repo or this PR...  Maybe they're created on demand, but I'm not seeing where...?

Reply:
hmm...I'm not sure why, it looks like they're here:
https://github.com/chapel-lang/chapel/tree/main/compiler/dyno/tools/chpldoc

Reply:
Oh, duh... I was looking at a repo that was a few days old, so didn't have them yet.  Thanks.